sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderClientIntegrationTest.java

```diff
@@ -104,6 +106,50 @@ void nonSessionMessageBatch(MessagingEntityType entityType) {
         }
     }
 
+    /**
+     * Verifies that we can schedule a message to a non-session entity.
+     */
+    @MethodSource("receiverTypesProvider")
+    @ParameterizedTest
+    void nonSessionScheduleMessage(MessagingEntityType entityType) {
+        // Arrange
+        setSenderAndReceiver(entityType);
+
+        final Instant scheduledEnqueueTime = Instant.now().plusSeconds(10);
+        final String messageId = UUID.randomUUID().toString();
+        final String contents = "Some-contents";
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(contents, messageId);
+
+        // Act
+        long sequenceNumber = sender.scheduleMessage(message, scheduledEnqueueTime);
+
+        // Assert
+        Assertions.assertTrue(sequenceNumber > 0);
```

Comment:
The sequence number can be 0 if it is in the beginning of stream.

Reply:
I am changing it to >=0 but When I try to send first message to a Queue, it always start  from 1.  But  SB docs says that  after reaching to max, it can roll over to zero.